docs/sources/articles/basics.md

```diff
@@ -35,21 +35,19 @@ and download it from [Docker Hub](https://hub.docker.com) to a local
 image cache.
 
 > **Note**:
-> When the image has successfully downloaded, you will see a 12 character
+> When the image has been successfully downloaded, you will see a 12 character
```

Comment:
When the image is successfully downloaded, you see...

![image](https://cloud.githubusercontent.com/assets/1347209/7917573/39c0c5ac-0840-11e5-8b11-59c0f206fa32.png)